internal/broker/bind_create_test.go

```diff
@@ -272,6 +277,123 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		}`, fixture.PlanId, customExpirationSeconds), t)
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
+	t.Run("should return 404 for not existing binding", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		path := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+
+		// when
+		response := CallAPI(httpServer, http.MethodGet, path, "", t)
+
+		// then
+		require.Equal(t, http.StatusNotFound, response.StatusCode)
+	})
+
+	t.Run("should return created kubeconfig", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		path := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+		body := fmt.Sprintf(`
+		{
+			"service_id": "123",
+			"plan_id": "%s",
+			"parameters": {	
+				"service_account": true	
+				}	
+		}`, fixture.PlanId)
+
+		// when
+		response := CallAPI(httpServer, http.MethodPut, path, body, t)
+		require.Equal(t, http.StatusCreated, response.StatusCode)
+
+		response = CallAPI(httpServer, http.MethodGet, path, "", t)
+
+		// then
+		require.Equal(t, http.StatusOK, response.StatusCode)
+
+		binding := unmarshal(t, response)
+
+		credentials, ok := binding.Credentials.(map[string]interface{})
+		require.True(t, ok)
+		kubeconfig := credentials["kubeconfig"].(string)
+
+		duration, err := getTokenDuration(t, kubeconfig)
+		require.NoError(t, err)
+		assert.Equal(t, expirationSeconds*time.Second, duration)
+
+		//// verify connectivity using kubeconfig from the generated binding
+		newClient := kubeconfigClient(t, kubeconfig)
+		_, err = newClient.CoreV1().Secrets("default").Get(context.Background(), "secret-to-check-first", v1.GetOptions{})
+		assert.NoError(t, err)
+	})
+
+	t.Run("should return selected binding when multiple bindings created", func(t *testing.T) {
```

Comment:
Please add test case creating two bindings for the same instance and getting the first one.

Reply:
Please add test case creating two bindings with the same bindingID and two different instanceIDs.
What is the expected behaviour?